src/CMakeLists.txt

```diff
@@ -637,9 +637,14 @@ install(TARGETS ceph-mon DESTINATION bin)
 if(NOT ALLOCATOR STREQUAL "jemalloc")
   set(disable_jemalloc "DISABLE_JEMALLOC=1")
 endif()
+set(EXTRA_CXXFLAGS -fPIC -Wno-unused-variable -DNDEBUG)
+if(FREEBSD)
+  set(EXTRA_CXXFLAGS ${EXTRA_CXXFLAGS} -Irocksdb/include -I/usr/local/include)
+  set(MAKEFLAGS ${MAKEFLAGS})
```

Comment:
@tchaikov 
Right.... Turns out that that whole block is bogus. It stems back from the time that I also had rocksdb installed as package. And then includefiles started to be mixed. And linking was a mess as well.
Looks like things have improved on all ends.
So the only things really required to be set are NDEBUG and CXX.
I'll fix.